pkg/notifications/client.go

```diff
@@ -77,3 +77,34 @@ func SetEmptyToNil(value string) *string {
 	}
 	return &value
 }
+
+// SendTemplatesNotification - Sends a notification about a template to the patch service
+func SendTemplatesNotification(orgID string, eventName EventName, templates []api.TemplateResponse) {
+	if config.Get().TemplatesNotificationsClient != nil && len(templates) > 0 {
+		eventNameStr := eventName.String()
+		newUUID, _ := uuid.NewRandom()
+		e := cloudevents.NewEvent()
+		e.SetSource("urn:redhat:source:console:app:repositories")
+		e.SetID(newUUID.String())
+		e.SetType("com.redhat.console.repositories." + eventNameStr)
+		e.SetSubject("urn:redhat:subject:console:rhel:" + eventNameStr)
+		e.SetTime(time.Now())
+		e.SetExtension("redhatorgid", orgID)
```

Comment:
These fields are set for the repository notifications. Not sure it's super important here, but I've copied them over as they seem like they might contain useful information.